src/main/java/com/yahoo/bullet/pubsub/memory/MemoryPubSub.java

```diff
@@ -0,0 +1,80 @@
+/*
+ *  Copyright 2017, Yahoo Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.pubsub.memory;
+
+import com.yahoo.bullet.common.BulletConfig;
+import com.yahoo.bullet.pubsub.PubSub;
+import com.yahoo.bullet.pubsub.PubSubException;
+import com.yahoo.bullet.pubsub.Publisher;
+import com.yahoo.bullet.pubsub.Subscriber;
+import lombok.extern.slf4j.Slf4j;
+import org.asynchttpclient.AsyncHttpClient;
+import org.asynchttpclient.AsyncHttpClientConfig;
+import org.asynchttpclient.DefaultAsyncHttpClient;
+import org.asynchttpclient.DefaultAsyncHttpClientConfig;
+
+import java.util.Collections;
+import java.util.List;
+
+@Slf4j
+public class MemoryPubSub extends PubSub {
+    private static final int NO_TIMEOUT = -1;
+    public static final int OK_200 = 200;
```

Comment:
You don't need  to define these final vars here. Just use https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/Response.Status.html.